packages/core/src/utils/sourceMaps.ts

```diff
@@ -0,0 +1,21 @@
+import type { ExceptionStackFrame } from '..';
+import { globalObject } from '../globalObject';
+
+export const cachedBundleIdStackFrameMap = new Map<string, ExceptionStackFrame[]>();
```

Comment:
Can you add a test to verify that this works as a cache indeed? Something like

1. Have a function that throws an error
2. Throw error twice
3. Verify cache size the same

I am fearing this can run the client out of memory if a lot of errors happen, in which case we might need to add an invalidation technique